data/macos.yaml

```diff
@@ -11,6 +11,19 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#System_Logs'
 ---
+name: MacOSAppleUnifiedLog
+doc: Apple Unified Log
+sources:
+- type: FILE
+  attributes:
+    paths:
+    - '/var/db/diagnostics/Persist/logdata.Persistent.YYYYMMDD THHMMSS.tracev3'
```

Comment:
I assume `YYYYMMDD THHMMSS` should be a wild card? also is the space in the string?

Reply:
Correct, I will change "YYYYMMDD THHMMSS" with "*".
The space was a typo

Reply:
Know that "YYYYMMDDTHHMMSS" will result in an exact string match

not a glob "[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]T[0-9][0-9][0-9][0-9][0-9][0-9]"

Reply:
Fixed

Reply:
As mentioned in my comments on the PR #298 , I'm referring specifically to the log files *.tracev3 and the resource files under uuidtext folder (this leaves out the log statistics file for example, which are in the same folder). would you prefer to keep them anyway?